### PR TITLE
docs: vuex README grammar suggestions

### PR DESCRIPTION
**Pull request by barriebyron**
Wording/content questions:
line 61: let's not use a slash (/) to mean and/or, is it `signer`, `wallet`, or both? 
line 84: do we always call the API the `Cosmos API`? or is the correct name the `Cosmos SDK API`? 
all lines: @clockworkgr teach me! I am learning and want to hear if this PR was helpful, or how I can do better.

### CODE REVIEW

**Review by toschdev — APPROVED** [review]

File: packages/vuex/README.md [hunk 1]
```diff
@@ -137,23 +133,23 @@ await this.$store.dispatch('common/transfers/ServiceGetTxsEvent', {
...
+and then access the resulting state anywhere in your app like:
 
 ```
 this.$store.getters['common/transfers/getGetTxsEvent']({
   event: 'transfer.recipient%3D%27cosmos1xv9tklw7d82sezh9haa573wufgy59vmwe6xxe5%27'
 })
 ```
 
-You can read more about events you can query for [here](https://docs.tendermint.com/master/rpc/#/Websocket/subscribe).
+To learn more about the websocket events you can query, see the [Tendermint RPC Websocket](https://docs.tendermint.com/master/rpc/#/Websocket/subscribe) docs.
```
Comment (fadeev) [inline packages/vuex/README.md]:
```suggestion
To learn more about the WebSocket events you can query, see the [Tendermint RPC WebSocket](https://docs.tendermint.com/master/rpc/#/Websocket/subscribe) docs.
```

File: packages/vuex/README.md [hunk 2]
```diff
@@ -94,41 +96,35 @@ Registered as `common/env` provides basic environment setup, connection to API,
...
+Registered as `common/starport`, use this module if you are doing local development on a [Starport](http://github.com/tendermint/starport)-launched chain. This module configures the `env` module using the Starport `:12345/status` endpoint.
 
 ### Blocks
 
-Registered as `common/blocks`, this module will receive, store and decode the latest 20 blocks (configurable) as they appear in the websocket API.
-
-Provides:
-
-`getBlocks(no_of_latest_blocks_to_get)`
+Registered as `common/blocks`, this module receives, stores, and decodes the latest 20 blocks (configurable) as the blocks appear in the websocket API.
```
Comment (fadeev) [inline packages/vuex/README.md]:
```suggestion
Registered as `common/blocks`, this module receives, stores, and decodes the latest 20 blocks (configurable) as the blocks appear in the WebSocket API.
```

**Review by fadeev — APPROVED** [review]
Aside from me being a WebSocket police 😄 I think this is 👍

**Comment by barriebyron** [comment]
@fadeev we could use some WebSocket policing on https://docs.tendermint.com/master/rpc/#/Websocket/subscribe

**Comment by barriebyron** [comment]
what happens now? someone with write access will merge this PR? (automatic merge  or a person needs to merge?)

**Pull request merged**